src/DotNetWorker.Grpc/Handlers/InvocationHandler.cs

```diff
@@ -45,8 +46,9 @@ internal class InvocationHandler : IInvocationHandler
             _inflightInvocations = new ConcurrentDictionary<string, CancellationTokenSource>();
         }
 
-        public async Task<InvocationResponse> InvokeAsync(InvocationRequest request, bool enableUserException = false)
+        public async Task<InvocationResponse> InvokeAsync(InvocationRequest request, IOptions<WorkerOptions>? workerOptions = null)
         {
+            bool enableUserCodeException = workerOptions is not null && workerOptions.Value.EnableUserCodeException;
```

Comment:
Can also be done as so:

```suggestion
            bool enableUserCodeException = workerOptions?.Value.EnableUserCodeException ?? false;
```